migrations/env.py

```diff
@@ -25,7 +25,7 @@
 
 config.set_main_option(
     "sqlalchemy.url",
-    current_app.config.get("SQLALCHEMY_DATABASE_URI").replace("%", "%%"),
+    str(current_app.extensions["migrate"].db.engine.url).replace("%", "%%"),
```

Comment:
why this change @mtreacy002 ? just asking because at the moment this came to the project, I thought it was kind of auto-generated code 🤔 but i am not sure